seed/utils/inventory_filter.py

```diff
@@ -88,6 +88,12 @@ def get_filtered_results(request: Request, inventory_type: Literal['property', '
     if f'{inventory_type}_view_ids' in request.data and request.data[f'{inventory_type}_view_ids']:
         views_list = views_list.filter(id__in=request.data[f'{inventory_type}_view_ids'])
 
+    if request.query_params.get('ids_only') == 'true':
+        id_list = [view.id for view in views_list]
+        return JsonResponse({
+            'results': id_list
+        })
```

Comment:
I think it _might_ be more efficient to do something like this instead to get the ids -- I think this would make it happen at the DB level instead of evaluating the entire view in Python:
```
id_list = views_list.values_list('id', flat=True)
```

Also, I'm a bit conflicted about how this query param results in the pagination parameters being ignored... For our use case it definitely makes sense, but is a bit unexpected from an API perspective I think... Thoughts?

Reply:
Ah yeah, here's a note from the docs:
```
values() and values_list() are both intended as optimizations for a specific use case: retrieving a subset of data without the overhead of creating a model instance. This metaphor falls apart when dealing with many-to-many and other multivalued relations (such as the one-to-many relation of a reverse foreign key) because the “one row, one object” assumption doesn’t hold.
```
https://docs.djangoproject.com/en/4.0/ref/models/querysets/#django.db.models.query.QuerySet.values_list

Reply:
hmm... what if we renamed the query parameter to "all_ids_only" or something similar?

Reply:
Hmm, kind of lead me to believe it's not applying the filtering (which it still is). I think the least surprising option is to just paginate on this as well. On frontend we can pass some large page size number for this request, and/or loop until all pages are fetched

Another option could be to assert that page/page_size aren't passed if `ids_only=true`, return a 400 (bad request) and indicate in the message why they can't do that?